src/main/java/use_cases/PostManager.java

```diff
@@ -34,25 +51,70 @@ private boolean interactPost(Post targetPost, Object obj, boolean type){
         return false;
     }
 
+    /**
+     * Creates a post and adds it to the list of post
+     * @param owner who created the post
+     * @param postedTime the time it was posted
+     * @param recipe a recipe object containing the info of the recipe
+     * @param category which category it belongs
+     */
     public void createPost(User owner, LocalDateTime postedTime, Recipe recipe, String category)
     {
         Post p = new Post(owner.getId(), postedTime, recipe, category);
-        this.addPostToList(p);
+        this.createPost(p);
     }
+
+    /**
+     * Comments a post iff could be done returns true
```

Comment:
What does this mean? Do you mean "Comments on a Post iff interactPost returns true"?